app/controllers/conviction_approval_forms_controller.rb

```diff
@@ -27,4 +30,8 @@ def authorize_action(transient_registration)
   def update_conviction_sign_off
     @transient_registration.conviction_sign_offs.first.approve!(current_user)
   end
+
+  def success_path
```

Comment:
I don't think this is needed. Now we pass the value in as a hash, we have the key `success_path:` to tell us what the value relates to.